package.json

```diff
@@ -283,16 +283,20 @@
       "@fluentui/web-components/@storybook/html",
       "@fluentui/web-components/ts-loader",
       "@fluentui/web-components/ts-loader/**",
-      "@fluentui/web-components/webpack"
+      "@fluentui/web-components/ts-node",
+      "@fluentui/web-components/ts-node/**",
+      "@fluentui/web-components/webpack",
+      "@fluentui/react-18-tests-v9/react",
+      "@fluentui/react-18-tests-v9/react-dom"
     ]
   },
   "resolutions": {
     "@typescript-eslint/eslint-plugin": "4.22.0",
     "@typescript-eslint/experimental-utils": "4.22.0",
     "@typescript-eslint/parser": "4.22.0",
     "@types/jest-axe/axe-core": "4.2.1",
-    "@types/react": "16.14.23",
-    "@types/react-dom": "16.9.14",
+    "@types/react@>=16.0.0 <17.0.0": "@types/react@16.14.23",
```

Comment:
cant we have this only as `"@types/react@<18.0.0": "@types/react@16.14.23"`. ?

there is no 17 types present in the repo